src/main/java/com/metamx/metrics/JvmThreadsMonitor.java

```diff
@@ -0,0 +1,58 @@
+package com.metamx.metrics;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import java.util.Map;
+import org.gridkit.lab.jvm.perfdata.JStatData;
+import org.gridkit.lab.jvm.perfdata.JStatData.LongCounter;
+
+public class JvmThreadsMonitor extends AbstractMonitor
```

Comment:
On peak live threads: with the pattern that I suggest, peakLive seems to be a slightly more precise alternative to "simple live" metric, because the latter might miss thread creation burst and shutdown between doMonitor() calls.

And you will emit all the same info as currently + some extra info. You will be able to see "peak live without resetting" via "peak live with resetting" + Max aggregate in metrics dashboard.